admin/src/Components/KeyHandoutForm.js

```diff
@@ -177,24 +211,31 @@ class KeyHandoutForm extends React.Component {
             </div>
 
             <div>
-                <button className="uk-button uk-button-success uk-float-right" tabIndex="1" disabled={!can_save_member && !can_save_key}><i className="uk-icon-save"/> Spara</button>
+                <button className="uk-button uk-button-success uk-float-right" tabIndex="2" title="Spara ändringar" disabled={!can_save_member && !can_save_key}><i className="uk-icon-save"/> Spara</button>
+                {this.renderAccessyInviteSaveButton({has_signed, labaccess_enddate, pending_labaccess_days, member})}
             </div>
         </>;
-
+        
         return (
         <div className="meep">
-            <form className="uk-form" onSubmit={(e) => {e.preventDefault(); this.onSave(); return false;}}>
+            <form className="uk-form" onSubmit={(e) => {e.preventDefault(); this.save(); return false;}}>
                 <div className="uk-section">
                     <h2>1. Ta emot signerat labbmedlemsavtal</h2>
                     <p>Kontrollera att labbmedlemsavtalet är signerat och säkerställ att rätt medlemsnummer står väl synligt på labbmedlemsavtalet.</p>
                     <div>
                         <label htmlFor="signed"> 
-                            <input id="signed" style={{"verticalAlign": "middle"}} className="uk-checkbox" type="checkbox" tabIndex="1" checked={has_signed} onChange={(e) => this.signedChanged(e.target.checked)}/> &nbsp;
-                            Signerat labbmedlemsavtal mottaget.
+                            <input id="signed" style={{"verticalAlign": "middle"}} className="uk-checkbox" type="checkbox" tabIndex="1" checked={has_signed} onChange={(e) => {
+                                if (e.target.checked) {
+                                    member.labaccess_agreement_at = new Date().toISOString();
+                                } else {
+                                    member.labaccess_agreement_at = null;
```

Comment:
Removing the previous date should not be this simple. It should at least have a warning pop-up show since the date is completely wiped if it is saved, and this should most likely almost never be used.

Reply:
Ok, will have to add a ticket for this because it was already merged: https://github.com/makerspace/makeradmin/issues/242

I thought it was enough as it is because the save button is hidden when you uncheck it. But it is possible to change tab and save.

There should probably only be an alert if the date was read from the db, otherwise it will be annoying.